### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by zhengbli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,13 @@ To build the tool, simply clone the repo, and run `build.cmd` in command line or
 - `JS.fs`: handles the emitting of the `domWeb.js` and `domWindows.js`
 
 ## Contribution Guide
-The `dom.generated.d.ts` and `webworker.generated.d.ts` files that are used in TypeScript repo `master` branch are used as baselines. For each pull request, we will try run the script and compare the generated files with the baseline files. So to make the tests passed, please update the baseline and the code at the same pull request. 
+The `dom.generated.d.ts` and `webworker.generated.d.ts` files that are used in TypeScript repo `master` branch are used as baselines.
```
Comment (DanielRosenwasser) [inline README.md]: used in **the** TypeScript repository

I think you can remove "`master` branch" - it's mostly implied

File: README.md [hunk 2]
```diff
@@ -10,7 +10,13 @@ To build the tool, simply clone the repo, and run `build.cmd` in command line or
 - `JS.fs`: handles the emitting of the `domWeb.js` and `domWindows.js`
 
 ## Contribution Guide
-The `dom.generated.d.ts` and `webworker.generated.d.ts` files that are used in TypeScript repo `master` branch are used as baselines. For each pull request, we will try run the script and compare the generated files with the baseline files. So to make the tests passed, please update the baseline and the code at the same pull request. 
+The `dom.generated.d.ts` and `webworker.generated.d.ts` files that are used in TypeScript repo `master` branch are used as baselines. 
+For each pull request, we will try run the script and compare the generated files with the baseline files.
```
Comment (DanielRosenwasser) [inline README.md]: we will run the script

File: README.md [hunk 3]
```diff
@@ -10,7 +10,13 @@ To build the tool, simply clone the repo, and run `build.cmd` in command line or
 - `JS.fs`: handles the emitting of the `domWeb.js` and `domWindows.js`
 
 ## Contribution Guide
-The `dom.generated.d.ts` and `webworker.generated.d.ts` files that are used in TypeScript repo `master` branch are used as baselines. For each pull request, we will try run the script and compare the generated files with the baseline files. So to make the tests passed, please update the baseline and the code at the same pull request. 
+The `dom.generated.d.ts` and `webworker.generated.d.ts` files that are used in TypeScript repo `master` branch are used as baselines. 
+For each pull request, we will try run the script and compare the generated files with the baseline files. 
+So to make the tests passed, please update the baseline and the code at the same pull request.
```
Comment (DanielRosenwasser) [inline README.md]: In order to check whether the test passed, please update the baseline as well in any pull requests.

**Comment by zhengbli** [comment]
@DanielRosenwasser I updated accordingly. Do you have other comments?

**Comment by DanielRosenwasser** [comment]
:+1:

**Pull request merged**